build.sbt

```diff
@@ -85,7 +85,9 @@ lazy val root = (project in file("root"))
     riffRaffManifestProjectName := "media-service:media-atom-maker",
     riffRaffArtifactResources := Seq(
       (packageBin in Debian in app).value -> s"${(name in app).value}/${(name in app).value}.deb",
-      (packageBin in Universal in uploader).value -> s"${(name in uploader).value}/${(packageBin in Universal in uploader).value.getName}",
+      // we have an entry here for each lambda that uses this code (see the cloud formation)
+      (packageBin in Universal in uploader).value -> s"media-atom-uploader-s3-events/${(packageBin in Universal in uploader).value.getName}",
```

Comment:
Clunky but allows us to share a single build between the lambdas. 99% of the code will be shared, with only the entry points differing